binding/android/OrcaTestApp/orca-test-app/src/androidTest/java/ai/picovoice/orca/testapp/SentenceTests.java

```diff
@@ -1,5 +1,5 @@
 /*
-    Copyright 2024 Picovoice Inc.
+    Copyright 2025 Picovoice Inc.
```

Comment:
Double-check your copyright headers, doesn't look like all of them were updated
```suggestion
    Copyright 2024-2025 Picovoice Inc.
```

Reply:
This file is correct actually, since it didn't exist before 2025, but I will be sure to go through and check the rest of them!

Reply:
right, the diff made it look like it did